src/test/java/com/karankumar/bookproject/backend/utils/PredefinedShelfUtilsTest.java

```diff
@@ -121,4 +140,51 @@ void shouldGetAllBooksInChosenShelves() {
         Assertions.assertTrue(actualBooks.containsAll(expectedBooks));
     }
 
-}
+    @Test
+    void testAllPredefinedShelves() {
```

Comment:
Could you please rename this method to better reflect what is being tested? For example, `testValidPredefinedShelfNames` (or something better)?